modules/rtm/R/edr.wrapper.R

```diff
@@ -90,14 +91,21 @@ EDR <- function(paths,
 # Generate input files
     par.nir.lengths <- c(length(par.wl), length(nir.wl))
     cat(par.nir.lengths, file=file.path(output.path, "lengths.dat"), sep = ' ')
-    par.ind <- which(RT.matrix[,"wl"] %in% par.wl)    # PAR indices -- offset by 399
+    par.ind <- which(RT.matrix[,"wl"] %in% par.wl)    # PAR indices
     nir.ind <- which(RT.matrix[,"wl"] %in% nir.wl)    # NIR indices 
     cat(RT.matrix[par.ind,1], file = file.path(output.path, "reflect_par.dat"), sep=" ")
     cat(RT.matrix[nir.ind,1], file = file.path(output.path, "reflect_nir.dat"), sep=" ")
     cat(RT.matrix[par.ind,2], file = file.path(output.path, "trans_par.dat"), sep=" ")
     cat(RT.matrix[nir.ind,2], file = file.path(output.path, "trans_nir.dat"), sep=" ")
 # Call EDR -- NOTE that this requires that the ED2IN 
-    system(file.path(output.path, edr.exe.name), intern=TRUE)
+    albedo.files <- list.files(output.path, "albedo.*",full.names = TRUE)
```

Comment:
Good catch! This was left over from a past way of checking that it executed correctly, but I've since replaced it with a better way. I removed these two lines in the next commit (2163bd1).